### ISSUE TITLE
Bluetooth?

### ISSUE THREAD

**Issue by GameCake** [hengineer/CaptainsMess#5]
Hey, is this still WIFI only? :) We're just starting our BLE stuff now and were just curious where you'd gotten to.

**Comment by hengineer** [comment]
Yeah still just Wifi for now. I've been busy finishing the rest of the Spaceteam rebuild so I haven't had time to add Bluetooth yet.
